File: JavaScript/1013 - O Maior.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8').split(' ');
var a = parseInt(input[0]);
var b = parseInt(input[1]);
var c = parseInt(input[2]);

var maior = (a + b + Math.abs(a - b)) / 2;
maior = (maior + c + Math.abs(maior - c)) / 2;

console.log(`${maior} eh o maior`);

File: Clojure/1013 - O Maior.clj
(let [inputs (map #(Integer/parseInt %) (clojure.string/split (read-line) #"\s+"))
      a (nth inputs 0)
      b (nth inputs 1)
      c (nth inputs 2)
      maiorAB (int (/ (+ a b (Math/abs (- a b))) 2))]
  (if (> maiorAB c)
    (println (format "%d eh o maior" maiorAB))
    (println (format "%d eh o maior" c))))